data/json/recipes/recipe_others.json

```diff
@@ -671,16 +666,13 @@
   "skill_used": "survival",
   "time": 5000,
   "autolearn": true,
+  "using": [ [ "filament", 40 ] ],
   "qualities":[{"id":"HAMMER","level":1}],
   "components": [
     [[ "stick", 1 ], [ "2x4", 1 ]],
     [[ "steel_chunk", 1 ], [ "steel_lump", 1 ]],
     [
-      [ "cordage_short", 2, "LIST" ],
-      [ "sinew", 40 ],
-      [ "thread", 40 ],
-      [ "plant_fibre", 40 ],
-      [ "yarn", 40 ]
+      [ "cordage_short", 2, "LIST" ]
```

Comment:
I thank what your trying to do here is allow for cordage OR filament but how you have the recipe now is cordage AND filament. IDK if you would necessarily need both to make a makeshift hammer.

Reply:
Fixed.